witnet/assets/requests/price/crypto.js

```diff
@@ -211,7 +211,7 @@ module.exports = {
     WitnetRequestPriceFraxUsdt6: Witnet.PriceTickerRequest(
         sources, { 
             "gateapi.io/ticker": ["frax", "usdt"], 
-            "uniswap-v3/ticker": ["0xc2a856c3aff2110c1171b8f942256d40e980c726", "1"] 
+            // "uniswap-v3/ticker": ["0xc2a856c3aff2110c1171b8f942256d40e980c726", "1"] 
```

Comment:
The endpoint being used in the `uniswap-v3/ticker` [is not working](https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v3/graphql), although it is in [Uniswap's official documentation](https://docs.uniswap.org/api/subgraph/overview).

I'd comment the other price feed using `uniswap-v3/ticker`: [WitnetRequestPriceUlxUsdt6](https://github.com/witnet/witnet-feeds/pull/37/files#diff-95ce02e624e52d6d78a52a6a2a5aa069c01a6ce213ca5c7cb57c422ecdb8a2a8R472)